usr/local/www/status_dhcp_leases.php

```diff
@@ -186,9 +189,16 @@ function remove_duplicate($array, $field)
 				$f = $f+3;
 				break;
 			case "ends":
-				$leases[$l]['end'] = $data[$f+2];
-				$leases[$l]['end'] .= " " . $data[$f+3];
-				$f = $f+3;
+				if ($data[$f+1] == "never") {
+					// Quote from dhcpd.leases(5) man page:
+					// If a lease will never expire, date is never instead of an actual date.
+					$leases[$l]['end'] = $data[$f+1];
```

Comment:
It would be good to have a intl text for "Never" here, don't you think? gettext("Never")